shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/server/plugins/AnnotatedInputFormatDefinition.java

```diff
@@ -308,13 +309,15 @@ public RemoteReloader(Path fileName) {
 
       @Override
       protected Stream<Object> fetch(Instant lastUpdated) throws Exception {
-        if (config.isEmpty()) return Stream.empty();
+        if (config.isEmpty()) return Stream.empty(); // FIXME probably not OK
         final var url = config.get().getUrl();
         var response =
             Server.HTTP_CLIENT.send(
                 HttpRequest.newBuilder(URI.create(url)).GET().version(Version.HTTP_1_1).build(),
                 BodyHandlers.ofInputStream());
         try (var parser = RuntimeSupport.MAPPER.getFactory().createParser(response.body())) {
+          // TODO do i need to test that it's 2xx rather than specifically 200?
```

Comment:
Looking at the other 200-series codes, only 200 OK is guaranteed to return usable data.